Index: git-cola/cola/resources.py
===================================================================
--- git-cola.orig/cola/resources.py	2010-03-19 18:14:18.753586953 +0000
+++ git-cola/cola/resources.py	2010-03-19 18:16:53.013571380 +0000
@@ -23,7 +23,7 @@
 
 def html_docs():
     """Returns the path to the cola html documentation."""
-    return doc('git-cola.html')
+    return doc('git-cola.txt')
 
 def share(*args):
     """Returns a path relative to cola's /usr/share/ directory"""
