Index: git-cola-1.4.1.2/setup.py
===================================================================
--- git-cola-1.4.1.2.orig/setup.py	2010-01-03 15:23:07.416158955 +0000
+++ git-cola-1.4.1.2/setup.py	2010-01-03 15:27:55.780177641 +0000
@@ -3,6 +3,7 @@
 import os
 import sys
 import stat
+import shutil
 from glob import glob
 
 from distutils.core import setup
@@ -36,7 +37,7 @@
     # then try git-describe, then default.
     builtin_version = os.path.join('cola', 'builtin_version.py')
     if os.path.exists('version') and not os.path.exists(builtin_version):
-        shutils.copy('version', builtin_version)
+        shutil.copy('version', builtin_version)
 
     elif os.path.exists('.git'):
         version.write_builtin_version()
